elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/analyzer/impl/ElasticsearchAnalyzer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.analyzer.impl;
+
+import org.apache.lucene.analysis.Analyzer;
+
+/**
+ * Wraps analyzer implementations defined on Elasticsearch.
+ *
+ * @author Davide D'Alto
+ */
+public class ElasticsearchAnalyzer extends Analyzer {
+
+	private final String name;
+
+	public ElasticsearchAnalyzer(String name) {
+		this.name = name;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	@Override
+	protected TokenStreamComponents createComponents(String fieldName) {
+		return null;
```

Comment:
Yes, that's the issue I see with this. This "analyzer" will not be used
locally. So we should abstract and each backend could work with it own
implementation.